src/test/groovy/pl/noip/lolstats/lol/stats/jwt/JwtCheckerImplTest.groovy

```diff
@@ -3,11 +3,22 @@ package pl.noip.lolstats.lol.stats.jwt
 import io.jsonwebtoken.ExpiredJwtException
 import io.jsonwebtoken.MalformedJwtException
 import io.jsonwebtoken.SignatureException
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.boot.context.embedded.LocalServerPort
+import org.springframework.boot.test.context.SpringBootTest
+import org.springframework.test.context.ActiveProfiles
 import pl.noip.lolstats.lol.stats.time.TimeService
 import spock.lang.Specification
 
+
+@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT)
+@ActiveProfiles("test")
```

Comment:
why did u add that?
this is unit test, not integration test, it shouldn't use spring at all